### PR TITLE
Pinned to djangocms-attributes-field v0.1.0

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: HISTORY [hunk 1]
```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+1.1.2 (2016-06-29)
+------------------
+
+* Pinned to djangocms-attributes-field v0.1.0
+* Fixed model name used in migration
```
Comment (czpython) [inline HISTORY]: where? don't see this in changes

Comment (mkoistinen) [inline HISTORY]: This is a change that Yakky made in a previous commit, so, when I created this new changelog entry, I just added it.

**Comment by czpython** [comment]
👍  if ok for tests to fail?

**Comment by yakky** [comment]
Could #245 also be considered for 1.1.2?

**Comment by mkoistinen** [comment]
@yakky, @czpython please review again.

File: cmsplugin_filer_folder/cms_plugins.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ class FilerFolderPlugin(CMSPluginBase):
         fieldsets[0][1]['fields'].append('style')
 
     def get_folder_files(self, folder, user):
-        qs_files = folder.files.filter(image__isnull=True)
+        qs_files = folder.files.instance_of(File)
```
Comment (yakky) [inline cmsplugin_filer_folder/cms_plugins.py]: I'm pretty sure this will select images too

**Comment by mkoistinen** [comment]
Rebased.

File: HISTORY (hunk 1 above)
Comment (yakky) [inline HISTORY]: @mkoistinen could you add "* Fixed an issue with folder plugin in Django 1.9" (see #245)

**Comment by yakky** [comment]
LGTM

**Pull request merged**